avocado/utils/multipath.py

```diff
@@ -175,9 +173,10 @@ def get_path_status(disk_path):
 
 def fail_path(path):
     """
-    failing the individual paths
-    :param disk_path: disk path. Example: sda, sdb.
-    :return: True or False
+    Fail the individual paths
```

Comment:
I would recommend following PEP0257, for multi-line docstrings here and on other methods.

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Sphinx uses the first line for instance in some places to summarize methods and classes.

Reply:
@beraldoleal If I understood correctly you recommending to document all methods with a summary line + \s + description + \s + metadata?

Reply:
@wainersm kind of.

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. 

So yes, but only with **one** line between them.

On PEP, also says:

> The summary line may be on the same line as the opening quotes or on the next line.

So, here is a "personal taste". I like to write the summary line on the same line as the opening quotes to make it consistent with other docstrings that are not multi-line. So don't worry here. I will not block this PR if you decide to go with the second option.

Reply:
@wainersm sorry about the first comment, yes!!! "\n\n" == to only one empty line. :)

Reply:
@beraldoleal I've no preferences on this regard. However this module (and many others) places the summary on the next line of the opening quotes. So If I change this here the module will diverge from the pattern found on others. Also I would need to change the docstring of the whole module.
Considering that, I prefer to not introduce such as extensive changes here. Perhaps we can discuss the utils documentation on a blueprint, then settle on a standard and decide how to approach the existing modules?

Reply:
@wainersm , I was more concerned about the single line vs multi-line as described on pep257, where multi-line should have a single line followed by \n\n and the long description. About this IMO there is no much to discuss since we could follow PEP257 here. But yes, I understand that it might be too much change for this PR.

I will accept and submit the second round of changes after that.